app/src/main/java/com/github/se/icebreakrr/model/profile/ProfilesViewModel.kt

```diff
@@ -31,6 +38,9 @@ open class ProfilesViewModel(
   private val _error = MutableStateFlow<Exception?>(null)
   val error: StateFlow<Exception?> = _error
 
+  private val _tempProfilePictureBitmap = MutableStateFlow<Bitmap?>(null)
```

Comment:
Your implementation of a temporary local profile picture is smart! This indeed reduces the loading time.